tests/lib/rules/indent.js

```diff
@@ -5724,6 +5724,111 @@ ruleTester.run("indent", rule, {
             `,
             options: [4, { MemberExpression: 2 }],
             parserOptions: { ecmaVersion: 2015 }
+        },
+        {
+            code: unIndent`
+const foo = async (arg1,
```

Comment:
Since we are using `unIndent`, code in newly added tests can be indented like the code in all previous tests?